src/test/java/systemtests/EditCommandSystemTest.java

```diff
@@ -39,11 +39,8 @@
 import seedu.address.logic.commands.RedoCommand;
 import seedu.address.logic.commands.UndoCommand;
 import seedu.address.model.Model;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.person.*;
```

Comment:
Import statements have to be specific, wildcard usage is not allowed. Applies to all other classes that uses this as well.